lib/waterline/utils/query/process-all-records.js

```diff
@@ -228,15 +224,15 @@ module.exports = function processAllRecords(records, meta, modelIdentity, orm) {
           );
 
           if (!isProbablyValidPkValue) {
-            console.warn('\n'+
-              'Warning: Records sent back from a database adapter should always have a valid property\n'+
-              'that corresponds with the primary key attribute (`'+WLModel.primaryKey+'`).  But in this result set,\n'+
-              'after transforming columnNames back to attribute names for model `' + modelIdentity + '`,\n'+
-              'there is a record with a missing or invalid `'+WLModel.primaryKey+'`.\n'+
-              'Record:\n'+
-              '```\n'+
-              util.inspect(record, {depth:5})+'\n'+
-              '```\n'+
+            console.warn(
+              'Warning: Records sent back from a database adapter should always have a valid property '+
+              'that corresponds with the primary key attribute (`'+WLModel.primaryKey+'`).  But in this result set, '+
+              'after transforming columnNames back to attribute names for model `' + modelIdentity + '`, '+
+              'there is a record with a missing or invalid `'+WLModel.primaryKey+'`. '+
+              'Record: '+
+              '``` '+
```

Comment:
@alxndrsn `util.inspect` is going to add new lines here anyway, so I think in spots like this it may just make more sense to leave the lines before and after the backticks. Any thoughts on this?

Reply:
:man_shrugging: in which case I should probably just override `console.log`, `console.error` etc. :confused: